### PR TITLE
Fix some suspicious things in vendor

### PR DESCRIPTION
**Pull request by LK4D4**
I changed hash for gocapability and re-run vendor script. Also I change vet validation to use docker version, which run vet only on changed files to skip `golang/protobuf` warning.
We discovered those issues in #633

### CODE REVIEW

File: vendor/src/github.com/golang/protobuf/proto/all_test.go [hunk 1]
```diff
@@ -1281,9 +1281,7 @@ func TestEnum(t *testing.T) {
 // We don't care what the value actually is, just as long as it doesn't crash.
 func TestPrintingNilEnumFields(t *testing.T) {
 	pb := new(GoEnum)
-	if fmt.Sprintf("%+v", pb) == "" {
-		t.Errorf("expected non-empty string")
-	}
+	fmt.Sprintf("%+v", pb)
```
Comment (mrunalp) [inline vendor/src/github.com/golang/protobuf/proto/all_test.go]: Are we making a change or undoing something that was done earlier?

Comment (LK4D4) [inline vendor/src/github.com/golang/protobuf/proto/all_test.go]: Undoing, I think @avagin did this for passing CI and forgot about it.

Comment (mrunalp) [inline vendor/src/github.com/golang/protobuf/proto/all_test.go]: Okay, that makes sense :)

File: update-vendor.sh [hunk 2]
```diff
@@ -44,7 +44,7 @@ clone git github.com/codegangsta/cli 1.1.0
 clone git github.com/coreos/go-systemd v2
 clone git github.com/godbus/dbus v2
 clone git github.com/Sirupsen/logrus v0.7.3
-clone git github.com/syndtr/gocapability 8e4cdcb
+clone git github.com/syndtr/gocapability 66ef2aa
```
Comment (mrunalp) [inline update-vendor.sh]: Adding a pointer to https://github.com/docker/libcontainer/commit/c0640ae4610a32272e53e3964ca2699c89b04f04 for posterity :)

**Comment by mrunalp** [comment]
LGTM

**Comment by dqminh** [comment]
LGTM

**Pull request merged**